src/app/builder/builder.component.html

```diff
@@ -107,9 +111,16 @@
         </fieldset>
 
         <fieldset>
-          <img [src]="props.imageUrl || defaults.imageUrl" id="artwork-preview">
-          <label>Artwork URL</label>
-          <input type="text" [(ngModel)]="props.imageUrl" id="imageUrl" [placeholder]="defaults.imageUrl" name="imageUrl">
+          <img [src]="props.epImageUrl || defaults.epImageUrl" id="artwork-preview">
+          <label>Episode Artwork URL</label>
+          <p><small>This artwork will be featured prominently in the foreground of the player.</small></p>
+          <input type="text" [(ngModel)]="props.epImageUrl" id="epImageUrl" [placeholder]="defaults.epImageUrl" name="epImageUrl">
+        </fieldset>
+
+        <fieldset>
+          <label>Feed Artwork URL</label>
```

Comment:
Current behavior on the player: 

If only feed (and optional GUID) params are present:
- the `feedArtworkUrl`[ is pulled from the `itunes:image` of the feed](https://github.com/PRX/play.prx.org/blob/master/src/app/embed/adapters/feed.adapter.ts#L115) 
- the `artworkUrl` [is pulled from the `itunes:image` of the episode ](https://github.com/PRX/play.prx.org/blob/master/src/app/embed/adapters/feed.adapter.ts#L123)
- (note that the player already distinguishes between feed artwork for background and ep artwork for foreground; that's how the playlist feature currently allows the different episode images to rotate in as they are played, but keeps the background the same, [like this](http://play.prx.org/e?uf=http%3A%2F%2Ffeeds.getmortified.com%2FMortifiedPod&ge=http%3A%2F%2Fmortified.prx.org%2F%3Fp%3D525&gs=_blank&sp=10))
- if there is no episode artwork, then the `artworkUrl` will [fall back to match](https://github.com/PRX/play.prx.org/blob/master/src/app/embed/embed.component.ts#L77) the `feedArtworkUrl`

If override of artwork url param is present, it will [come through to the player](https://github.com/PRX/play.prx.org/blob/master/src/app/embed/adapters/qsd.adapter.ts#L23) as the `feedArtworkUrl`. 

If the user is making a play-most-recent-episode player, they don't [actually see the override options. ](https://github.com/PRX/play.prx.org/blob/master/src/app/builder/builder.component.html#L92)

This help clear things up?